assets/js/components/DeviceSizeTabBar.js

```diff
@@ -78,7 +78,7 @@ const DeviceSizeTabBar = ( {
 						{ icon }
 					</Tab>
 				);
-			}
+			},
```

Comment:
Huh, that's kinda weird. I get it, but it's unexpected to me...

Reply:
It's the `comma-dangle` rule. Just say the word and we can get rid of this rule @tofumatt !  

I don't like it, but I do like consistency and not thinking bout this kind of stuff :smile: +